scripts/toc.sh

```diff
@@ -5,8 +5,11 @@ set -o nounset
 if which "./scripts/gh-md-toc" >/dev/null 2>&1; then
   echo "Binary gh-md-toc exists."
 else
+  echo Downloading scripts
+  pushd ./scripts 
```

Comment:
use a different folder, probably `bin` and ignore that folder on git, currently downloading in scripts means it'll be committed

Reply:
Good suggestion. It is part of the gitignore right now. Using bin folder seems like good standard rather than juggling paths. Making fix.